DQMOffline/L1Trigger/python/L1TriggerDqmOffline_cff.py

```diff
@@ -254,3 +257,108 @@
 #l1EmulatorMonitorClient.remove(l1EmulatorQualityTests)
 l1EmulatorMonitorClient.remove(l1EmulatorErrorFlagClient)
 #l1EmulatorMonitorClient.remove(l1EmulatorEventInfoClient)
+
+#stage2 
+from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
+
+from L1Trigger.L1TGlobal.hackConditions_cff import *
+from L1Trigger.L1TMuon.hackConditions_cff import *
+from L1Trigger.L1TCalorimeter.hackConditions_cff import *
+
+from DQMOffline.L1Trigger.L1TStage2CaloLayer2Offline_cfi import *
+l1tStage2CaloLayer2OfflineDQMEmu.stage2CaloLayer2JetSource=cms.InputTag("valCaloStage2Layer2Digis")
+l1tStage2CaloLayer2OfflineDQMEmu.stage2CaloLayer2EtSumSource=cms.InputTag("valCaloStage2Layer2Digis")
+from DQMOffline.L1Trigger.L1TEGammaOffline_cfi import *
+l1tEGammaOfflineDQMEmu.stage2CaloLayer2EGammaSource=cms.InputTag("valCaloStage2Layer2Digis")
+
+from Configuration.StandardSequences.Eras import eras
+from DQM.L1TMonitor.L1TStage2_cff import *
+
+stage2UnpackPath = cms.Sequence(
+     l1tCaloLayer1Digis +
```

Comment:
For the moment, leave the packer in the sequence, and turn the LogError into LogInfo in case of no FED found, silenced after 5 occurrences.